src/sentry/relay/config/metric_extraction.py

```diff
@@ -205,6 +215,14 @@ def _get_alert_metric_specs(
     return specs
 
 
+@metrics.wraps("on_demand_metrics._get_alert_metric_specs")
+def _get_global_templates(project: Project) -> dict[str, GlobalTemplateOverride]:
+    templates: dict[str, GlobalTemplateOverride] = {}
+    if features.has("projects:span-metrics-extraction", project):
+        templates["spans_hardcoded"] = {"isEnabled": True}
```

Comment:
This template does not exist yet, but enabling it now will allow us to switch from project config to global config in a single Relay PR.